### PR TITLE
Added config node source nodelet indication

### PR DESCRIPTION
**Pull request by schmerl**
Config file now specifies if the node is a "normal" node or a "nodelet".

### CODE REVIEW

**Comment by schmerl** [comment]
Note that we could make `kind` optional and default to `NORMAL`.

File: src/rosdiscover/config.py [hunk 1]
```diff
@@ -12,6 +13,19 @@
 from .launch import Launch
 
 
+class ROSNodeKind(enum.Enum):
+    NORMAL = "normal"
```
Comment (ChrisTimperley) [inline src/rosdiscover/config.py:17]: maybe just `node` rather than `normal`?

**Review by ChrisTimperley — APPROVED** [review]
Looks good to me. I made a minor comment about node vs. normal.

**Comment by schmerl** [comment]
> 
> 
> Note that we could make `kind` optional and default to `NORMAL`.

Made this change - default is now `NODE`.

**Pull request merged**